Add prep content for self coordinate (#691)
CodeYourFuture/curriculum: common-content/en/module/how-our-curriculum-works/morning-orientation-block/index.md

+++
title="🎡 Morning orientation"
headless="true"
time=30
[objectives]
    1="Locate a morning orientation block"
    2="Check learning objectives for morning orientation block"
+++

> The morning orientation block is used to gather the community together and nominate a time-keeper and a facilitator (if they weren't already nominated during the week). The objectives of the morning orientation block are clear:

Reviewer: ```suggestion
We use the morning orientation block to gather the community together. We nominate a time-keeper and a facilitator (if they weren't already nominated during the week). The objectives of the morning orientation block are clear:
```
